app/helpers/audits_helper.rb

```diff
@@ -191,6 +191,28 @@ def audit_details(audit)
     "#{audit_user(audit)} #{audit_remote_address audit} #{audit_action_name audit} #{audited_type audit}: #{link_to(audit_title(audit), audit_path(audit))}".html_safe
   end
 
+  def nested_breadcrumbs
```

Comment:
this is probably my fault, but all helpers are included in every view (not sure why tbh) so, probably best to call it nested_host_audit_breadcrumbs or some other thing unique.